src/Subsystems/Intake.cpp

```diff
@@ -18,6 +18,8 @@ Intake::Intake() : Subsystem(kSubsystemName),
   rightMotor(RobotMap::kIDIntakeRight),
   leftMotor(RobotMap::kIDIntakeLeft) {
 
+  leftMotor.SetControlMode(frc::CANSpeedController::kFollower);
+  leftMotor.Set(RobotMap::kIDIntakeRight);
```

Comment:
I' going to fix this so it will correlate with the naming in the `feature/DesignUpdates.v1` branch so it will make merging this easier in the future. Although.... we will only have one motor in the main branch in the future so I don't entirely see the function of this PR